app/models/appointment.rb

```diff
@@ -36,6 +36,13 @@ def self.overdue
     where(status: 'scheduled').where('scheduled_date <= ?', Date.today)
   end
 
+  def self.appointments_per_facility
+    includes(:facility,
+             :patient => [:address,
```

Comment:
`includes(:patient)` (etc.) is irrelevant to this method. Let's let queries decide for themselves if they want to include patient/address/phone